analytics-core/src/main/java/com/liferay/analytics/android/api/interface/AnalyticsClient.kt

```diff
@@ -0,0 +1,28 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.analytics.android.api.`interface`
+
+import com.liferay.analytics.android.model.AnalyticsEventsMessage
+
+/**
+ * @author Igor Matos
+ * @author Allan Melo
+ */
+interface AnalyticsClient {
```

Comment:
Do we need an interface for AnalyticsClient? Can it have more than one implementations?

Reply:
Good point, we don't!